store/README.md

```diff
@@ -0,0 +1,10 @@
+# STORE
```

Comment:
You can, maybe, remove these files if they weren't required.
Also, you can maintain a `.gitkeep` file if you wish to keep these folders.

Reply:
done